conf/script/build-ha-csm

```diff
@@ -101,13 +101,14 @@ done
 argsfile=${1:-}
 
 if [[ -f $argsfile ]]; then
-    while IFS=': ' read name value; do
+    while IFS=': ' read name value || [[ $name =~ [a-zA-Z0-9] ]]; do
```

Comment:
"||" should be "&&"

$ while IFS=': ' read name value && [[ $name =~ [a-zA-Z0-9] ]]; do echo "hello"; done
xyz
hello
<space><enter>
$

Reply:
No, it is if `read` fails (returns false) then we have the other condition. Tested this on the setup.